ci/values-gateway.yaml

```diff
@@ -8,3 +9,9 @@ clusterRole:
 presets:
   logsCollection:
     enabled: false
+config:
+  processors:
+    batch:
+      send_batch_max_size: 50
+      send_batch_size: 10
```

Comment:
how did we arrive at these values ? This seems to too low

Reply:
This was from before I added replicas, I was seeing 413 errors. I removed this and am not seeing 413 anymore 👍